auth-service/build.gradle

```diff
@@ -82,6 +82,7 @@ jacocoTestReport {
 		classDirectories.setFrom(files(classDirectories.files.collect {
 			fileTree(dir: it, exclude: [
 					"com/petclinic/**/AuthServiceApplication.class",
+					"**/*MapperImpl.class"
```

Comment:
Why is this class excluded from the jacoco report? Is it because it's automatically generated?

Reply:
Yes.
I still wrote relevant mapper tests.
From my research, neither JaCoCo nor MapStruct expose a preferable solution to this